src/codegen/cfg.rs

```diff
@@ -420,6 +427,61 @@ impl ControlFlowGraph {
         }
         self.blocks[self.current].add(ins);
     }
+    // Function to modify array length temp
```

Comment:
Use three slashes for doc comments. In addition, please add a few words about what your function does and in what context.